finalPrep/annotations.js

```diff
@@ -2,41 +2,54 @@
 // Use the double // to create a new comment
```

Comment:
I left a couple notes but overall VERY strong annotations, great use of technical vocabulary 🚀